sdk/OpenTap.Sdk.MSBuild/InstallOpenTapPackages.cs

```diff
@@ -119,6 +120,88 @@ public void Cancel()
         {
             cts.Cancel();
         }
+
+        /// <summary> Load Authentication tokens from Items into settings. </summary>
+        /// <returns>True if successful.</returns>
+        bool ProcessAuthTokens()
```

Comment:
I feel like this is encouraing people to put user tokens directly in their .csproj / check it into source control. I know we will use environment variables for this, but I don't feel great about it.